resources/core/charts/api-controller/templates/tests/test-api-controller.yaml

```diff
@@ -16,6 +24,9 @@ spec:
     env:
     - name: DOMAIN_NAME
       value: {{ .Values.global.domainName }}
+    env:
```

Comment:
All env variables must be in the same `env` section

Reply:
Another error brought to you by too much copy&paste :)